--- tests/Broadcasting/PusherBroadcasterTest.php	2019-08-25 22:34:15.362149700 +0200
+++ tests/Broadcasting/PusherBroadcasterTest2.php	2019-08-25 22:34:09.428000000 +0200
@@ -158,7 +158,7 @@
                 ->with('callback', false)
                 ->andReturn(false);
 
-        $user = m::mock('User');
+        $user = m::mock(\Illuminate\Contracts\Auth\Authenticatable::class);
         $user->shouldReceive('getAuthIdentifier')
              ->andReturn(42);
 
