--- tests/Cache/CacheMemcachedConnectorTest.php	2019-08-25 22:50:50.666064500 +0200
+++ tests/Cache/CacheMemcachedConnectorTest2.php	2019-08-25 22:51:52.624182500 +0200
@@ -93,7 +93,7 @@
 
     protected function memcachedMockWithAddServer($returnedVersion = [])
     {
-        $memcached = m::mock(stdClass::class);
+        $memcached = m::mock(\stdClass::class);
         $memcached->shouldReceive('addServer')->once()->with($this->getHost(), $this->getPort(), $this->getWeight());
         $memcached->shouldReceive('getServerList')->once()->andReturn([]);
 
