--- tests/Support/SupportCollectionTest.php	2019-08-21 00:23:32.000000000 +0200
+++ tests/Support/SupportCollectionTest2.php	2019-08-21 00:23:25.000000000 +0200
@@ -3186,7 +3186,8 @@
         $collection = new Collection;
         $this->expectException(Exception::class);
         $this->expectExceptionMessage('Property [foo] does not exist on this collection instance.');
-        $collection->foo;
+
+        $var = $collection->foo;
     }
 
     public function testGetWithNullReturnsNull()
