--- tests/Database/DatabaseEloquentBelongsToManyWithDefaultAttributesTest.php	2019-08-25 23:18:25.727335500 +0200
+++ tests/Database/DatabaseEloquentBelongsToManyWithDefaultAttributesTest2.php	2019-08-25 23:19:06.604071700 +0200
@@ -26,10 +26,10 @@
         $relation = $this->getMockBuilder(BelongsToMany::class)->setMethods(['touchIfTouching'])->setConstructorArgs($this->getRelationArguments())->getMock();
         $relation->withPivotValue(['is_admin' => 1]);
 
-        $query = m::mock(stdClass::class);
+        $query = m::mock(\stdClass::class);
         $query->shouldReceive('from')->once()->with('club_user')->andReturn($query);
         $query->shouldReceive('insert')->once()->with([['club_id' => 1, 'user_id' => 1, 'is_admin' => 1]])->andReturn(true);
-        $relation->getQuery()->shouldReceive('getQuery')->andReturn($mockQueryBuilder = m::mock(stdClass::class));
+        $relation->getQuery()->shouldReceive('getQuery')->andReturn($mockQueryBuilder = m::mock(\stdClass::class));
         $mockQueryBuilder->shouldReceive('newQuery')->once()->andReturn($query);
 
         $relation->attach(1);
