--- tests/Database/DatabaseEloquentMorphToManyTest.php	2019-08-25 22:26:20.036816300 +0200
+++ tests/Database/DatabaseEloquentMorphToManyTest2.php	2019-08-25 22:27:08.656765600 +0200
@@ -7,6 +7,7 @@
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Builder;
 use Illuminate\Database\Eloquent\Relations\MorphToMany;
+use stdClass;
 
 class DatabaseEloquentMorphToManyTest extends TestCase
 {
