--- tests/Database/DatabaseSoftDeletingTraitTest.php	2019-08-25 22:50:51.296041900 +0200
+++ tests/Database/DatabaseSoftDeletingTraitTest2.php	2019-08-25 23:03:47.326562900 +0200
@@ -18,7 +18,7 @@
     {
         $model = m::mock(DatabaseSoftDeletingTraitStub::class);
         $model->makePartial();
-        $model->shouldReceive('newModelQuery')->andReturn($query = m::mock(stdClass::class));
+        $model->shouldReceive('newModelQuery')->andReturn($query = m::mock(\stdClass::class));
         $query->shouldReceive('where')->once()->with('id', '=', 1)->andReturn($query);
         $query->shouldReceive('update')->once()->with([
             'deleted_at' => 'date-time',
