--- tests/Database/DatabaseConnectionTest.php	2019-08-25 22:50:50.887040700 +0200
+++ tests/Database/DatabaseConnectionTest2.php	2019-08-25 22:59:39.312999400 +0200
@@ -33,7 +33,7 @@
     public function testSettingDefaultCallsGetDefaultGrammar()
     {
         $connection = $this->getMockConnection();
-        $mock = m::mock(stdClass::class);
+        $mock = m::mock(\stdClass::class);
         $connection->expects($this->once())->method('getDefaultQueryGrammar')->will($this->returnValue($mock));
         $connection->useDefaultQueryGrammar();
         $this->assertEquals($mock, $connection->getQueryGrammar());
@@ -42,7 +42,7 @@
     public function testSettingDefaultCallsGetDefaultPostProcessor()
     {
         $connection = $this->getMockConnection();
-        $mock = m::mock(stdClass::class);
+        $mock = m::mock(\stdClass::class);
         $connection->expects($this->once())->method('getDefaultPostProcessor')->will($this->returnValue($mock));
         $connection->useDefaultPostProcessor();
         $this->assertEquals($mock, $connection->getPostProcessor());
