--- vendor/aws/aws-sdk-php/src/S3/S3EndpointMiddleware.php	2019-08-25 12:07:23.278963600 +0200
+++ vendor/aws/aws-sdk-php/src/S3/S3EndpointMiddleware2.php	2019-08-25 12:07:17.914000000 +0200
@@ -117,34 +117,34 @@
         CommandInterface $command,
         RequestInterface $request
     ) {
-        $accelerate = isset($command['@use_accelerate_endpoint'])
-            ? $command['@use_accelerate_endpoint'] : $this->accelerateByDefault;
-        $dualStack = isset($command['@use_dual_stack_endpoint'])
-            ? $command['@use_dual_stack_endpoint'] : $this->dualStackByDefault;
-        $pathStyle = isset($command['@use_path_style_endpoint'])
-            ? $command['@use_path_style_endpoint'] : $this->pathStyleByDefault;
-
-        if ($accelerate && $dualStack) {
-            // When try to enable both for operations excluded from s3-accelerate,
-            // only dualstack endpoints will be enabled.
-            return $this->canAccelerate($command)
-                ? self::ACCELERATE_DUALSTACK
-                : self::DUALSTACK;
-        }
-
-        if ($accelerate && $this->canAccelerate($command)) {
-            return self::ACCELERATE;
-        }
-
-        if ($dualStack) {
-            return self::DUALSTACK;
-        }
-
-        if (!$pathStyle
-            && self::isRequestHostStyleCompatible($command, $request)
-        ) {
-            return self::HOST_STYLE;
-        }
+        // $accelerate = isset($command['@use_accelerate_endpoint'])
+        //     ? $command['@use_accelerate_endpoint'] : $this->accelerateByDefault;
+        // $dualStack = isset($command['@use_dual_stack_endpoint'])
+        //     ? $command['@use_dual_stack_endpoint'] : $this->dualStackByDefault;
+        // $pathStyle = isset($command['@use_path_style_endpoint'])
+        //     ? $command['@use_path_style_endpoint'] : $this->pathStyleByDefault;
+
+        // if ($accelerate && $dualStack) {
+        //     // When try to enable both for operations excluded from s3-accelerate,
+        //     // only dualstack endpoints will be enabled.
+        //     return $this->canAccelerate($command)
+        //         ? self::ACCELERATE_DUALSTACK
+        //         : self::DUALSTACK;
+        // }
+
+        // if ($accelerate && $this->canAccelerate($command)) {
+        //     return self::ACCELERATE;
+        // }
+
+        // if ($dualStack) {
+        //     return self::DUALSTACK;
+        // }
+
+        // if (!$pathStyle
+        //     && self::isRequestHostStyleCompatible($command, $request)
+        // ) {
+        //     return self::HOST_STYLE;
+        // }
 
         return self::PATH_STYLE;
     }
@@ -158,9 +158,9 @@
     private function getBucketStyleHost(CommandInterface $command, $host)
     {
         // For operations on the base host (e.g. ListBuckets)
-        if (!isset($command['Bucket'])) {
-            return $host;
-        }
+        // if (!isset($command['Bucket'])) {
+        //     return $host;
+        // }
 
         return "{$command['Bucket']}.{$host}";
     }
