--- vendor/aws/aws-sdk-php/src/Middleware.php	2019-08-25 12:09:02.966489400 +0200
+++ vendor/aws/aws-sdk-php/src/Middleware2.php	2019-08-25 12:17:25.760968800 +0200
@@ -345,15 +345,15 @@
                 return $handler($command, $request)
                     ->then(
                         function (ResultInterface $res) use ($start) {
-                            if (!isset($res['@metadata'])) {
-                                $res['@metadata'] = [];
-                            }
-                            if (!isset($res['@metadata']['transferStats'])) {
-                                $res['@metadata']['transferStats'] = [];
-                            }
+                            // if (!isset($res['@metadata'])) {
+                            //     $res['@metadata'] = [];
+                            // }
+                            // if (!isset($res['@metadata']['transferStats'])) {
+                            //     $res['@metadata']['transferStats'] = [];
+                            // }
 
-                            $res['@metadata']['transferStats']['total_time']
-                                = microtime(true) - $start;
+                            // $res['@metadata']['transferStats']['total_time']
+                            //     = microtime(true) - $start;
 
                             return $res;
                         },
