--- vendor/aws/aws-sdk-php/src/S3/S3Client.php	2019-08-25 12:09:03.468487900 +0200
+++ vendor/aws/aws-sdk-php/src/S3/S3Client2.php	2019-08-25 12:21:08.232184500 +0200
@@ -452,11 +452,11 @@
                 CommandInterface $command,
                 RequestInterface $request = null
             ) use ($handler) {
-                if ($command->getName() === 'HeadObject'
-                    && !isset($command['@http']['decode_content'])
-                ) {
-                    $command['@http']['decode_content'] = false;
-                }
+                // if ($command->getName() === 'HeadObject'
+                //     && !isset($command['@http']['decode_content'])
+                // ) {
+                //     $command['@http']['decode_content'] = false;
+                // }
 
                 return $handler($command, $request);
             };
@@ -495,21 +495,21 @@
                                 ['CommonPrefixes', 'Prefix'],
                             ];
 
-                            foreach ($topLevel as $key) {
-                                if (isset($result[$key])) {
-                                    $result[$key] = urldecode($result[$key]);
-                                }
-                            }
-                            foreach ($nested as $steps) {
-                                if (isset($result[$steps[0]])) {
-                                    foreach ($result[$steps[0]] as $key => $part) {
-                                        if (isset($part[$steps[1]])) {
-                                            $result[$steps[0]][$key][$steps[1]]
-                                                = urldecode($part[$steps[1]]);
-                                        }
-                                    }
-                                }
-                            }
+                            // foreach ($topLevel as $key) {
+                            //     if (isset($result[$key])) {
+                            //         $result[$key] = urldecode($result[$key]);
+                            //     }
+                            // }
+                            // foreach ($nested as $steps) {
+                            //     if (isset($result[$steps[0]])) {
+                            //         foreach ($result[$steps[0]] as $key => $part) {
+                            //             if (isset($part[$steps[1]])) {
+                            //                 $result[$steps[0]][$key][$steps[1]]
+                            //                     = urldecode($part[$steps[1]]);
+                            //             }
+                            //         }
+                            //     }
+                            // }
 
                         }
 
