--- vendor/filp/whoops/src/Whoops/Util/Misc.php	2019-08-07 11:00:00.000000000 +0200
+++ vendor/filp/whoops/src/Whoops/Util/Misc.php	2019-08-16 12:11:04.000000000 +0200
@@ -47,14 +47,14 @@
      */
     public static function translateErrorCode($error_code)
     {
-        $constants = get_defined_constants(true);
-        if (array_key_exists('Core', $constants)) {
-            foreach ($constants['Core'] as $constant => $value) {
-                if (substr($constant, 0, 2) == 'E_' && $value == $error_code) {
-                    return $constant;
-                }
-            }
-        }
+        // $constants = get_defined_constants(true);
+        // if (array_key_exists('Core', $constants)) {
+        //     foreach ($constants['Core'] as $constant => $value) {
+        //         if (substr($constant, 0, 2) == 'E_' && $value == $error_code) {
+        //             return $constant;
+        //         }
+        //     }
+        // }
         return "E_UNKNOWN";
     }
     
