--- vendor/phpunit/phpunit/src/Util/Configuration.php	2019-08-11 08:56:55.000000000 +0200
+++ vendor/phpunit/phpunit/src/Util/Configuration.php	2019-08-15 17:34:13.000000000 +0200
@@ -125,7 +125,7 @@
     {
         $result = [];

-        foreach ($this->xpath->query('extensions/extension') as $extension) {
+        foreach ($this->xpath->query('phpunit/extensions/extension') as $extension) {
             \assert($extension instanceof DOMElement);

             $class     = (string) $extension->getAttribute('class');
@@ -160,7 +160,7 @@
         $excludeDirectory                   = [];
         $excludeFile                        = [];

-        $tmp = $this->xpath->query('filter/whitelist');
+        $tmp = $this->xpath->query('phpunit/filter/whitelist');

         if ($tmp->length === 1) {
             if ($tmp->item(0)->hasAttribute('addUncoveredFilesFromWhitelist')) {
@@ -182,19 +182,19 @@
             }

             $includeDirectory = $this->readFilterDirectories(
-                'filter/whitelist/directory'
+                'phpunit/filter/whitelist/directory'
             );

             $includeFile = $this->readFilterFiles(
-                'filter/whitelist/file'
+                'phpunit/filter/whitelist/file'
             );

             $excludeDirectory = $this->readFilterDirectories(
-                'filter/whitelist/exclude/directory'
+                'phpunit/filter/whitelist/exclude/directory'
             );

             $excludeFile = $this->readFilterFiles(
-                'filter/whitelist/exclude/file'
+                'phpunit/filter/whitelist/exclude/file'
             );
         }

@@ -219,7 +219,7 @@
      */
     public function getGroupConfiguration(): array
     {
-        return $this->parseGroupConfiguration('groups');
+        return $this->parseGroupConfiguration('phpunit/groups');
     }

     /**
@@ -227,7 +227,7 @@
      */
     public function getTestdoxGroupConfiguration(): array
     {
-        return $this->parseGroupConfiguration('testdoxGroups');
+        return $this->parseGroupConfiguration('phpunit/testdoxGroups');
     }

     /**
@@ -237,7 +237,7 @@
     {
         $result = [];

-        foreach ($this->xpath->query('listeners/listener') as $listener) {
+        foreach ($this->xpath->query('phpunit/listeners/listener') as $listener) {
             \assert($listener instanceof DOMElement);

             $class     = (string) $listener->getAttribute('class');
@@ -268,7 +268,7 @@
     {
         $result = [];

-        foreach ($this->xpath->query('logging/log') as $log) {
+        foreach ($this->xpath->query('phpunit/logging/log') as $log) {
             \assert($log instanceof DOMElement);

             $type   = (string) $log->getAttribute('type');
@@ -342,7 +342,7 @@
             'request'      => [],
         ];

-        foreach ($this->xpath->query('php/includePath') as $includePath) {
+        foreach ($this->xpath->query('phpunit/php/includePath') as $includePath) {
             $path = (string) $includePath->textContent;

             if ($path) {
@@ -350,7 +350,7 @@
             }
         }

-        foreach ($this->xpath->query('php/ini') as $ini) {
+        foreach ($this->xpath->query('phpunit/php/ini') as $ini) {
             \assert($ini instanceof DOMElement);

             $name  = (string) $ini->getAttribute('name');
@@ -359,7 +359,7 @@
             $result['ini'][$name]['value'] = $value;
         }

-        foreach ($this->xpath->query('php/const') as $const) {
+        foreach ($this->xpath->query('phpunit/php/const') as $const) {
             \assert($const instanceof  DOMElement);

             $name  = (string) $const->getAttribute('name');
@@ -369,7 +369,7 @@
         }

         foreach (['var', 'env', 'post', 'get', 'cookie', 'server', 'files', 'request'] as $array) {
-            foreach ($this->xpath->query('php/' . $array) as $var) {
+            foreach ($this->xpath->query('phpunit/php/' . $array) as $var) {
                 \assert($var instanceof DOMElement);

                 $name     = (string) $var->getAttribute('name');
@@ -897,10 +897,10 @@
      */
     public function getTestSuiteConfiguration(string $testSuiteFilter = ''): TestSuite
     {
-        $testSuiteNodes = $this->xpath->query('testsuites/testsuite');
+        $testSuiteNodes = $this->xpath->query('phpunit/testsuites/testsuite');

         if ($testSuiteNodes->length === 0) {
-            $testSuiteNodes = $this->xpath->query('testsuite');
+            $testSuiteNodes = $this->xpath->query('phpunit/testsuite');
         }

         if ($testSuiteNodes->length === 1) {
