--- vendor/phpunit/php-file-iterator/src/Factory.php	2019-08-15 18:18:55.000000000 +0200
+++ vendor/phpunit/php-file-iterator/src/Factory.php	2019-08-15 18:19:06.000000000 +0200
@@ -18,9 +18,9 @@
      * @param array|string $prefixes
      * @param array        $exclude
      *
-     * @return \AppendIterator
+     * @return Iterator
      */
-    public function getFileIterator($paths, $suffixes = '', $prefixes = '', array $exclude = []): \AppendIterator
+    public function getFileIterator($paths, $suffixes = '', $prefixes = '', array $exclude = []): Iterator
     {
         if (\is_string($paths)) {
             $paths = [$paths];
@@ -45,12 +45,12 @@
             }
         }

-        $iterator = new \AppendIterator;
+        // $iterator = new \AppendIterator;

         foreach ($paths as $path) {
             if (\is_dir($path)) {
-                $iterator->append(
-                    new Iterator(
+                // $iterator->append(
+                    return new Iterator(
                         $path,
                         new \RecursiveIteratorIterator(
                             new \RecursiveDirectoryIterator($path, \RecursiveDirectoryIterator::FOLLOW_SYMLINKS | \RecursiveDirectoryIterator::SKIP_DOTS)
@@ -58,12 +58,12 @@
                         $suffixes,
                         $prefixes,
                         $exclude
-                    )
-                );
+                    );
+                // );
             }
         }

-        return $iterator;
+        // return $iterator;
     }

     protected function getPathsAfterResolvingWildcards(array $paths): array
@@ -72,9 +72,9 @@

         foreach ($paths as $path) {
             if ($locals = \glob($path, GLOB_ONLYDIR)) {
-                $_paths = \array_merge($_paths, \array_map('\realpath', $locals));
+                $_paths = \array_merge($_paths, $locals);
             } else {
-                $_paths[] = \realpath($path);
+                $_paths[] = $path;
             }
         }

