--- src/Illuminate/Foundation/Application.php	2019-10-28 22:21:05.124170000 +0100
+++ src/Illuminate/Foundation/Application2.php	2019-10-28 22:22:09.577705600 +0100
@@ -138,6 +138,11 @@
     protected $namespace;
 
     /**
+     * @var LaravelApp
+     */
+    protected $laravelApp;
+
+    /**
      * Create a new Illuminate application instance.
      *
      * @param  string|null  $basePath
@@ -152,6 +157,29 @@
         $this->registerBaseBindings();
         $this->registerBaseServiceProviders();
         $this->registerCoreContainerAliases();
+
+        $this->laravelApp = new class extends \Laravel\Sdk\LaravelApp
+        {
+            public function GetVersion()
+            {
+                return Application::VERSION;
+            }
+
+            public function InternalRegisterController($name, $controller)
+            {
+                app()->bind($name, $controller);
+            }
+
+            public function InternalSetUserModel($userModel)
+            {
+            }
+        };
+
+        global $peachpie_laravel_loader;
+
+        if ($peachpie_laravel_loader) {
+            $peachpie_laravel_loader->InternalAppStarted($this->laravelApp);
+        }
     }
 
     /**
