--- src/Illuminate/Foundation/Application.php	2019-10-28 22:24:16.955162900 +0100
+++ src/Illuminate/Foundation/Application2.php	2019-10-28 22:25:01.393104900 +0100
@@ -158,6 +158,10 @@
         $this->registerBaseServiceProviders();
         $this->registerCoreContainerAliases();
 
+        if (!class_exists("\\Laravel\\Sdk\\LaravelApp")) {
+            return;
+        }
+
         $this->laravelApp = new class extends \Laravel\Sdk\LaravelApp
         {
             public function GetVersion()
