--- src/Illuminate/Routing/RouteAction.php	2019-08-16 11:59:10.000000000 +0200
+++ src/Illuminate/Routing/RouteAction.php	2019-08-16 11:52:22.000000000 +0200
@@ -28,17 +28,17 @@
         // If the action is already a Closure instance, we will just set that instance
         // as the "uses" property, because there is nothing else we need to do when
         // it is available. Otherwise we will need to find it in the action list.
-        if (is_callable($action, true)) {
-            return ! is_array($action) ? ['uses' => $action] : [
-                'uses' => $action[0].'@'.$action[1],
-                'controller' => $action[0].'@'.$action[1],
-            ];
-        }
+        // if (is_callable($action, true)) {
+        //     return ! is_array($action) ? ['uses' => $action] : [
+        //         'uses' => $action[0].'@'.$action[1],
+        //         'controller' => $action[0].'@'.$action[1],
+        //     ];
+        // }

         // If no "uses" property has been set, we will dig through the array to find a
         // Closure instance within this list. We will set the first Closure we come
         // across into the "uses" property that will get fired off by this route.
-        elseif (! isset($action['uses'])) {
+        /*else*/if (! isset($action['uses'])) {
             $action['uses'] = static::findCallable($action);
         }

