--- src/Illuminate/Hashing/ArgonHasher.php	2019-08-22 15:21:16.753188900 +0200
+++ src/Illuminate/Hashing/ArgonHasher2.php	2019-08-22 15:35:31.858339200 +0200
@@ -60,7 +60,7 @@
      */
     public function make($value, array $options = [])
     {
-        $hash = password_hash($value, $this->algorithm(), [
+        $hash = \password_hash($value, $this->algorithm(), [
             'memory_cost' => $this->memory($options),
             'time_cost' => $this->time($options),
             'threads' => $this->threads($options),
@@ -109,7 +109,7 @@
      */
     public function needsRehash($hashedValue, array $options = [])
     {
-        return password_needs_rehash($hashedValue, $this->algorithm(), [
+        return \password_needs_rehash($hashedValue, $this->algorithm(), [
             'memory_cost' => $this->memory($options),
             'time_cost' => $this->time($options),
             'threads' => $this->threads($options),
