--- src/Illuminate/Hashing/BcryptHasher.php	2019-08-22 15:21:16.758175000 +0200
+++ src/Illuminate/Hashing/BcryptHasher2.php	2019-08-22 15:36:36.293907900 +0200
@@ -44,7 +44,7 @@
      */
     public function make($value, array $options = [])
     {
-        $hash = password_hash($value, PASSWORD_BCRYPT, [
+        $hash = \password_hash($value, PASSWORD_BCRYPT, [
             'cost' => $this->cost($options),
         ]);
 
@@ -83,7 +83,7 @@
      */
     public function needsRehash($hashedValue, array $options = [])
     {
-        return password_needs_rehash($hashedValue, PASSWORD_BCRYPT, [
+        return \password_needs_rehash($hashedValue, PASSWORD_BCRYPT, [
             'cost' => $this->cost($options),
         ]);
     }
