--- src/Illuminate/Database/Seeder.php	2019-08-21 00:30:07.000000000 +0200
+++ src/Illuminate/Database/Seeder2.php	2019-08-21 00:30:01.000000000 +0200
@@ -122,4 +122,14 @@
                     ? $this->container->call([$this, 'run'])
                     : $this->run();
     }
+
+    public function Invoke()
+    {
+        return $this->__invoke();
+    }
+
+    public function ToPhpValue()
+    {
+        return $this;
+    }
 }
