--- src/Illuminate/Support/Str.php	2019-08-15 22:42:07.000000000 +0200
+++ src/Illuminate/Support/Str.php	2019-08-15 23:16:08.000000000 +0200
@@ -466,7 +466,7 @@
         $title = str_replace('@', $separator.'at'.$separator, $title);

         // Remove all characters that are not the separator, letters, numbers, or whitespace.
-        $title = preg_replace('![^'.preg_quote($separator).'\pL\pN\s]+!u', '', static::lower($title));
+        // $title = preg_replace('![^'.preg_quote($separator).'\pL\pN\s]+!u', '', static::lower($title));

         // Replace all separator characters and whitespace by a single separator
         $title = preg_replace('!['.preg_quote($separator).'\s]+!u', $separator, $title);
